### PR TITLE
Updating datashader notebooks to remove InteractiveImage

### PR DESCRIPTION
**Pull request by jlstevens**
This PR transfers the works from https://github.com/pyviz/datashader/pull/751 as those topic notebooks have been removed from datashader.

### CODE REVIEW

File: nyc_taxi/nyc_taxi-nongeo.ipynb [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    "labelled.redim.range(tip_amount=(0,60), passenger_count=(-0.5,6.5))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we've reduced the resolution along the x axis so that instead of getting isolated points for this inherently discrete data, you can see more-visible horizontal line segments.\n",
+    "\n",
+    "The above plots all use Bokeh directly, but a much wider range of interactive plots can be built easily using the separate [HoloViews](http://holoviews.org) library, which builds Bokeh and Matplotlib plots from high-level specifications.  For instance, Datashader currently only provides 2D aggregates, but you can easily make a zoomable one-dimensional histogram using HoloViews to dynamically collapse across a second dimension:"
```
Comment (jbednar) [inline nyc_taxi/nyc_taxi-nongeo.ipynb]: This comment is out of date, but presumably you're relying on there being a whole pass done once pyviz-topics is complete?

File: opensky.ipynb [hunk 2]
```diff
@@ -310,98 +289,27 @@
...
-    "# Remove unused columns and declare categoricals\n",
-    "flightpaths = flightpaths[['longitude', 'latitude', 'origin', 'ascending', 'velocity']]\n",
-    "flightpaths['origin']    = flightpaths.origin.astype('category')\n",
-    "flightpaths['ascending'] = flightpaths.ascending.astype('bool')\n",
-    "\n",
-    "# Export to Parquet\n",
-    "args = dict(engine=\"fastparquet\", compression=\"snappy\", has_nulls=False, write_index=False)\n",
-    "flightpaths.to_parquet(\"../data/opensky.parq\", **args)\n",
-    "```"
+    "This data was obtained by running a cron job with the following script running at one-minute intervals over a four-day period using the script [on the wiki](https://github.com/pyviz/datashader/wiki/OpenSky-download-script)\n"
```
Comment (jbednar) [inline opensky.ipynb]: Presumably the script shouldn't be on the wiki, but in the project dir, but it looks like this .ipynb doesn't have a directory yet?  If so the script should be moved into the directory once it exists.

File: opensky.ipynb [hunk 3]
```diff
@@ -310,98 +289,27 @@
...
    ]
   }
  ],
  "metadata": {
+  "kernelspec": {
```
Comment (jbednar) [inline opensky.ipynb]: Should be cleared.

File: opensky.ipynb (hunk 2 above)
Comment (jlstevens) [reply opensky.ipynb]: Agreed.

**Comment by jsignell** [comment]
@jlstevens are you going to make these changes or do you want me to take over?

**Comment by jlstevens** [comment]
Thanks for merging! I was done with the code updates.

There are also Jim's comments that need addressing but I suspect these notebooks will need another pass anyway in a different PR.

**Comment by jsignell** [comment]
Yep yep!

**Pull request merged**